src/chef_workstation.js

```diff
@@ -117,6 +124,32 @@ function getUserConfig() {
   return userConfigCache;
 }
 
+function getAppConfig() {
+  if (loadedAppConfig == null && appConfigCache == null) {
+    try {
+      loadedAppConfig = TOML.parse(fs.readFileSync(appConfigFile));
+      appConfigCache = loadedAppConfig;
+    } catch(error) {
+      appConfigCache = {};
+    }
+  }
+  return appConfigCache;
+}
+
+function saveAppConfig() {
+  if ((loadedAppConfig != null && appConfigCache != null) || (loadedAppConfig == null && appConfigCache != {})) {
+    try {
+      if (!fs.existsSync(ws_dir)) {
```

Comment:
Good catch - it's likely that the App will be run before `chef-run` does any setup.

Reply:
yep ^ that.